### PR TITLE
fix: remove `*` imports

### PR DESCRIPTION
**Pull request by Samoed**
## Checklist

- [X] Run tests locally to make sure nothing is broken using `make test`. 
- [X] Run the formatter to format the code using `make lint`. 

Ref https://github.com/embeddings-benchmark/mteb/issues/1463
Also merged changes from `main` and found 3 datasets that previously never imported:
- Ddisco
- GeorgianSentimentClassification
- WongnaiReviewsClassification

### CODE REVIEW

**Comment by isaac-chung** [comment]
Nice! Was wondering actually, if this could be merged to main instead? I don't think there were any major compatible changes? (as in, everything should run the same as before)

File: mteb/__init__.py [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+    AbsTaskMultilabelClassification,
+    AbsTaskPairClassification,
+    AbsTaskReranking,
+    AbsTaskRetrieval,
+    AbsTaskSpeedTask,
+    AbsTaskSTS,
+    AbsTaskSummarization,
+    MultilingualTask,
+)
 from mteb.benchmarks.benchmarks import (
```
Comment (KennethEnevoldsen) [inline mteb/__init__.py:20]: We could remove these

**Review by KennethEnevoldsen — COMMENTED** [review]
Looks great. I'm very happy about this!

It was a big frustration for me in #1567 so I am very happy to see it. Did you do it all manually?

> Nice! Was wondering actually, if this could be merged to main instead? I don't think there were any major compatible changes? (as in, everything should run the same as before)

Before you could e.g. do 

```
from mteb import load_datasets
```

I believe this will no longer be possible

**Comment by isaac-chung** [comment]
I see a script being used to generate these.  To see changes from this PR, I was switching the commits one at a time. 

Might be nice if the merge from main was separated.

**Comment by Samoed** [comment]
Yes, as @isaac-chung mentioned, I wrote [a script](https://github.com/embeddings-benchmark/mteb/blob/07f139159a19bd10c78b484e5550e8d624deca49/scripts/generate_imports.py) to generate imports for tasks, but for other directories, I did it manually.

**Review by isaac-chung — APPROVED** [review]
Thanks for tackling this :) 
Might be worth running the same benchmarks in https://github.com/embeddings-benchmark/mteb/issues/1463 again as a comparison.

**Comment by Samoed** [comment]
Here is profile results of profiling, but it is hard to tell difference 
profile

**Pull request merged**